### PR TITLE
Handling errors in Python SDK

### PR DESCRIPTION
**Pull request by paras-n**

### CODE REVIEW

File: src/extractpdf/extract_txt_from_pdf_error_case.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Adobe. All rights reserved.
```
[the remark this answers is not in the document]
Comment (varkumaradobe) [reply src/extractpdf/extract_txt_from_pdf_error_case.py:1]: Also check whole copyright content

File: src/extractpdf/extract_txt_from_pdf_error_case.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+from adobe.pdfservices.operation.exception.exceptions import ServiceApiException, ServiceUsageException, SdkException
+from adobe.pdfservices.operation.pdfops.options.extractpdf.extract_pdf_options import ExtractPDFOptions
+from adobe.pdfservices.operation.pdfops.options.extractpdf.extract_element_type import ExtractElementType
+from adobe.pdfservices.operation.execution_context import ExecutionContext
+from adobe.pdfservices.operation.io.file_ref import FileRef
+from adobe.pdfservices.operation.pdfops.extract_pdf_operation import ExtractPDFOperation
+
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+
+
```
[the remark this answers is not in the document]
Comment (paras-n) [reply src/extractpdf/extract_txt_from_pdf_error_case.py:25]: Please refer to the attached documentation: https://cs.stanford.edu/people/nick/py/python-style1.html#:~:text=PEP8%20requires%202%20blank%20lines,has%203%20lines%20between%20functions.

File: src/extractpdf/extract_txt_from_pdf_error_case.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+def handle_exception(exception_type, exception_message):
+    logging.info(exception_type)
+    logging.info(exception_message)
+    return exception_type, exception_message
+
+
```
[the remark this answers is not in the document]
Comment (paras-n) [reply src/extractpdf/extract_txt_from_pdf_error_case.py:31]: Please refer to the attached documentation: https://cs.stanford.edu/people/nick/py/python-style1.html#:~:text=PEP8%20requires%202%20blank%20lines,has%203%20lines%20between%20functions.

File: src/extractpdf/extract_txt_from_pdf_error_case.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        with_client_id(os.getenv('PDF_SERVICES_CLIENT_ID')). \
+        with_client_secret(os.getenv('PDF_SERVICES_CLIENT_SECRET')). \
+        build()
+
+    # Create an ExecutionContext using credentials and create a new operation instance.
+    execution_context = ExecutionContext.create(credentials)
+    extract_pdf_operation = ExtractPDFOperation.create_new()
+
+    # Set operation input from a source file.
+    source = FileRef.create_from_local_file(base_path + "/resources/" + input_file_name)
```
[the remark this answers is not in the document]
Comment (paras-n) [reply src/extractpdf/extract_txt_from_pdf_error_case.py:48]: We wanted to generalize this sample to handle all types of exceptions(as discussed internally). Hence, we have taken the file name as argument.

File: src/extractpdf/extract_txt_from_pdf_error_case.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (paras-n) [reply src/extractpdf/extract_txt_from_pdf_error_case.py:1]: Sure

**Review by varkumaradobe — APPROVED** [review]
Comments are resolved, approving based on internal PR: https://git.corp.adobe.com/dc/dc-cpf-python-sdk-samples/pull/83

**Review by varkumaradobe — APPROVED** [review]
LGTM, file name changed to replect errors.

**Pull request merged**